### ISSUE TITLE
PInout

### ISSUE THREAD

**Issue by pricard32** [atsage/node-red-contrib-PN532#2]
Can you attach a pinout schematic to explain how to connect it to a RPI ?
I can't make it to work.

**Comment by alexmicuplusfour** [comment]
@pricard32 I managed to get it working with this guide http://wiki.sunfounder.cc/index.php?title=PN532_NFC_Module_for_Raspberry_Pi

**Comment by MasterOfPuppets111** [comment]
> @pricard32 I managed to get it working with this guide http://wiki.sunfounder.cc/index.php?title=PN532_NFC_Module_for_Raspberry_Pi

Do you use i2c or spi? Does node work with this guide or only standalone?

**Comment by alexmicuplusfour** [comment]
> > @pricard32 I managed to get it working with this guide http://wiki.sunfounder.cc/index.php?title=PN532_NFC_Module_for_Raspberry_Pi
> 
> Do you use i2c or spi? Does node work with this guide or only standalone?

SPI. I'm using it with node-red.
![screenshot-192 168 100 36_1880-2021 01 06-15_31_26](https://user-images.githubusercontent.com/37256676/103773837-589ae600-5034-11eb-8d17-23f973180cdc.png)

**Comment by MasterOfPuppets111** [comment]
Thank you very much for your answer  :).
Did you do all the steps in the guide (install libnfc, write the configuration file) or did you only wire it up an install  the PN532 node in node red and it works?

**Comment by alexmicuplusfour** [comment]
I did all the steps.
make sure you set the tiny switches on the board to the correct position.
![2021-01-06 20_16_39-pi@raspberrypi_ _etc_nfc](https://user-images.githubusercontent.com/37256676/103805411-3917b380-505c-11eb-8981-535be32dc89e.png)
![foto_no_exif](https://user-images.githubusercontent.com/37256676/103805812-cb1fbc00-505c-11eb-8b1b-3800d9628a8a.jpg)

**Comment by atsage** [comment]
I'm using I2C for less wires.  The code was written for I2C, but I don't remember much about it.  Not sure how @alexmicuplusfour got it working with SPI unless he made a fork.  As he said, make sure the switches are set correctly.

blob:https://imgur.com/b8131bb6-6dae-43ef-8b99-eba5a77a97e2
blob:https://imgur.com/2fcfa679-42af-4f9c-bffc-c5036290118a

**Comment by asage-me** [comment]
I'm using I2C for less wires. The code was written for I2C, but I don't remember much about it. Not sure how @alexmicuplusfour got it working with SPI unless he made a fork. As he said, make sure the switches are set correctly.

![20210106_135941](https://user-images.githubusercontent.com/30732752/103810124-b0cbeb00-5028-11eb-8f75-923e9389a6d3.jpg)
![20210106_140244](https://user-images.githubusercontent.com/30732752/103810133-b45f7200-5028-11eb-97cb-6c770b7b10a7.jpg)

**Comment by asage-me** [comment]
Yeah, so the readme is not correct.  I will update soon.  Basically you want to follow the wiki page @alexmicuplusfour posted and it will work with whatever connection method you use since the libnfc library works with both.
